resources/windows/windows.pkr.hcl

```diff
@@ -81,9 +109,18 @@ build {
         Name    = source.value.name
         Version = local.version
       }
+      run_tags = {
+        Name    = source.value.name
+        Version = local.version
+      }
     }
   }
 
+  provisioner "powershell" {
```

Comment:
Moving this a bit earlier so that we can WinRM with the install creds if need be (without using the `-debug` flag)